submissions/solaryasha/a-tiny-js-world-without-oop/index.js

```diff
@@ -0,0 +1,77 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+    species: 'dog',
+    name: 'Buddy',
+    gender: 'male',
+    legs: 4,
+    hands: 0,
+    saying: "woof-woof"
+};
+const cat = {
+    species: 'cat',
+    name: 'Daisy',
+    gender: 'female',
+    legs: 4,
+    hands: 0,
+    saying: "meow-meow"
+};
+const man = {
+    species: 'human',
+    name: 'Jack',
+    gender: 'male',
+    legs: 2,
+    hands: 2,
+    saying: "What's up?"
+};
+const woman = {
+    species: 'human',
+    name: 'Ann',
+    gender: 'female',
+    legs: 2,
+    hands: 2,
+    saying: 'Have a nice day'
+};
+const catWoman = {
+    species: 'half human half cat',
+    name: 'Selina',
+    gender: 'female',
+    legs: 2,
+    hands: 2,
+    saying: cat.saying
+};
+
+function convertObject(obj) {
```

Comment:
Function name is too generic. The function composes inhabitant's profile. Smth like `composeProfile` or `getInhabitantProfile` would be a more explaining function name.
Same is applicable to parameter name. `obj` is too generic. `inhabitant` perhaps would be more expressive.